vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/SheetFilterTableIT.java

```diff
@@ -35,4 +36,24 @@ public void filter_removeTable_hideFilter() {
         spreadsheet.getContextMenu().getItem("Delete Table B2:F6").click();
         waitUntil(arg0 -> !cell.hasPopupButton());
     }
+
+    @Test
+    public void sheetWithFilterTable_rowIsRemoved_filterOptionsAvailable() {
+        selectRegion("A1", "A5");
+        final var cell = getCellAt(1, 1);
+        cell.contextClick();
+        clickItem("Create Table on A1:A5");
```

Comment:
nit: this test could also use the `SpreadsheetTable` fixture (the same setup as the test above)

Reply:
Oh, right, that's better. Refactored.